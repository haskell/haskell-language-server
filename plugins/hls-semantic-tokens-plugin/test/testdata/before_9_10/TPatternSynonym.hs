{-# LANGUAGE PatternSynonyms #-}
module TPatternSynonym where


pattern Foo = 1


