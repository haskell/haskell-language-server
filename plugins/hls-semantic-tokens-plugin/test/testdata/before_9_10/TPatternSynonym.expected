5:9-12 TPatternSynonym "Foo"
